9 Types/Types.playground/Pages/Beyond The Standard Library.xcplaygroundpage/Contents.swift

```diff
@@ -17,3 +17,4 @@
  
 [Previous](@previous)  |  page 10 of 13  |  [Next: Importing Frameworks](@next)
 */
+TrainingShoe = 12, "Nike", "released 2014", "$150"
```

Comment:
Do you know what `TrainingShoe` contains, is it `12`, `"Nike"`, `"$150"` or all of them?

If it's all of them, how do you access the individual elements in `TrainingShoe`?